cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,72 @@
-describe('', () => {
-  before(() => {
+/// <reference types="cypress" />
+/// <reference types="../support" />
+describe('article flow', () => {
+  let article;
 
+  beforeEach(() => {
+    cy.loginConduit();
+
+    cy.task('newArticle')
+      .then((newArticle) => {
+        article = newArticle;
+        cy.createArticle(newArticle);
+      }
+      );
+    cy.visit('/editor');
   });
+  it('should allow to create an article', () => {
+    cy.findByPlaceholder('Article Title')
+      .type(article.title);
+
+    cy.findByPlaceholder('What\'s this article about?')
+      .type(article.description);
+
+    cy.findByPlaceholder('Write your article (in markdown)')
+      .type(article.body);
 
-  it('', () => {
+    cy.findByPlaceholder('Enter tags')
+      .type(article.tag);
 
+    setTimeout(() => {
+      cy.contains('button', 'Publish Article')
+        .click();
+    }, '1000');
+
+    cy.url()
+      .should('contain', '/article');
```

Comment:
It would be nice to check if there is slug of the article you published in the url